src/bp/ui-studio/src/web/views/FlowBuilder/common/routing.tsx

```diff
@@ -0,0 +1,114 @@
+import classnames from 'classnames'
+import _ from 'lodash'
+import Mustache from 'mustache'
+import React, { Component } from 'react'
+import { OverlayTrigger, Popover, Well } from 'react-bootstrap'
```

Comment:
It's existing code that i've copy-pasted, since we're gonna change all that soon I didn't bother to convert it. It keeps the same look as old nodes (since they are temporary)

Reply:
ok cool but let's add a TODO because we're going to migrate the styling to blueprint before we get rid of the nodes